package.json

```diff
@@ -23,19 +23,19 @@
     "optimization"
   ],
   "engines": {
-    "node": ">=10"
+    "node": "^10 || ^12 || >=14"
```

Comment:
@XhmikosR: Do you think we need a major version bump for this one?

Reply:
Technically, yeah. Even though I don't think people should use non-LTS releases, this has happened before with autoprefixer or a dep it was depending on. Hence why I didn't touch postcss in this PR :)